CHANGELOG.md

```diff
@@ -28,7 +28,8 @@ _TBD_
 ### Removed
 - [x] Removed deprecated Plan properties: `expiredTime`, `recurringAmount`, `trialAmount`, `setupAmount`,
       `recurringPeriodUnit`, `recurringPeriodLength`, `recurringPeriodLimit`, `trialPeriodUnit`, `trialPeriodLength`,
-      `contractTermUnit`, `contractTermLength`, `minQuantity`, `maxQuantity` 
+      `contractTermUnit`, `contractTermLength`, `minQuantity`, `maxQuantity`
+- [x] Removed LeadSources for Transactions, Invoices and Subscriptions
```

Comment:
> Changelogs are for humans, not machines. 

Would be nice to format changelog for better reading. Classes, methods, properties like `code`, the names and titles which need to emphasize is nice to read as `bold` text